wiki/osu!_Program_Files/en.md

```diff
@@ -1,113 +1,119 @@
 
 ### Screenshots
 
-**Screenshots (F12 by default) of osu!**. The saved screenshot's file extension (.jpg/.png) is based on what you set on the Options menu.
+**This folder holds screenshots the player has created in osu!**. The saved screenshot's file extension (.jpg/.png) is based on what the player set on the Options menu.
+
+To create a screenshot, press the screenshot key (F12 by default).
 
-The format is `screenshot###` where "###" is the screenshot number count.
+The file name structure is `screenshot###`, where "###" is the screenshot number count.
 
 ### Skins
 
+<<<<<<< HEAD
+This folder holds **user-created skins, which can be used to customise the in-game interface.** Players can download skins from the [Skinning subforum](https://osu.ppy.sh/community/forums/15). Players can install skins by double-clicking on the skin from a file manager. 
+=======
 This folder holds **user-created skins, which can be used to customise the in-game interface.** You can download skins from the [Skinning subforum](https://osu.ppy.sh/community/forums/15). You can install .osk skins by double-clicking them. If the downloaded skin is in folder form, you will have to place the folder here yourself. If it is in a ZIP or RAR format, you must extract it first. You can change your skins at [Options menu under Skins tab (Skin Selector)](/wiki/Options). Please bear in mind that the selected skin is only visible to you.
+>>>>>>> e78270d1d920905df8e145d2cccd9e5ab8864daa
 
-For further reference, please refer to the [Skinning](/wiki/Skinning) page. Also, "osu! by peppy" is the only skin without its folder and cannot be deleted directly.
+For further reference, please refer to the [Skinning](/wiki/Skinning) page. Also, "osu! by peppy" is the only skin without its folder and cannot be deleted.
 
 ### Songs
 
-**Your compendium of osu! beatmaps**. Usually contains .osu (difficulties), .mp3/.ogg (music file), .jpg/.png/.gif (BG image), .osb (SB file) and .mp4/.flv (video file). May also contains .wav/.ogg (hitsound file) and folders (SB spites and/or skin).
+**This folder holds the player's osu! beatmaps**. Usually contains .osu (difficulties), .mp3/.ogg (music files), .jpg/.png/.gif (background images), .osb (storyboard files) and .mp4/.flv (video files). May also contains .wav/.ogg (hitsound files) and folders (storyboard spites and/or skin).
 
-The format is `{Beatmap number} {Artist} - {Song Title}`.
+The file name structure is `{Beatmap number} {Artist} - {Song Title}`.
 **Example:** [57950 SOUND HOLIC - Drive My Life](https://osu.ppy.sh/beatmapsets/57950)
 
 Please note that some very old beatmaps (for example, [Kenji Ninuma - DISCO PRINCE](https://osu.ppy.sh/beatmapsets/1) or [Dudelstudios - Angry Video Game Nerd Theme [MATURE CONTENT]](https://osu.ppy.sh/beatmapsets/66)), as well as unsubmitted beatmaps, do not follow the format.
 
-### Hidden Folders
+## Hidden folders
```

Comment:
"Hidden folders" needs a short description

also the rest of these sections need more information in order to justify splitting them up into separate sections. Otherwise, the information should be condensed into paragraph or two

Reply:
Added a short description to the Hidden folders in cd34b18